codespell_lib/data/dictionary.txt

```diff
@@ -20640,6 +20640,7 @@ indepenendence->independence
 indepenent->independent
 indepenently->independently
 indepent->independent
+indepentend->independent
```

Comment:
I would suggest adding
```
indepentendly->independently
```

I can see a few occurrences of this misspelling in GitHub.